scylla-cql/src/frame/value.rs

```diff
@@ -101,6 +104,42 @@ impl CqlVarint {
     }
 }
 
+impl CqlVarint {
+    fn as_normalized_slice(&self) -> &[u8] {
+        let digits = self.0.as_slice();
+        if digits.is_empty() {
+            return digits;
+        }
```

Comment:
`num_bigint::BigInt` seems to convert empty slices into a zero, see here: https://docs.rs/num-bigint/latest/src/num_bigint/bigint/convert.rs.html#385. So, 

```
CqlVarint::from_signed_bytes_be(vec![]) != CqlVarint::from_signed_bytes_be(vec![0])
```
but

```
BigInt::from_signed_bytes_be(vec![]) == BigInt::from_signed_bytes_be(vec![0])
```

I think it would make the structure more well-behaved if we exactly preserve properties such as equality or ordering when we map to BigInt, so here I think we should `return &[0]`.